packages/project-access/src/project/cap.ts

```diff
@@ -425,7 +426,9 @@ async function loadCdsModuleFromProject(capProjectPath: string, strict: boolean
     if (global) {
         global.cds = cds;
     }
-
+    // correct cds.env for current project root. Especially needed CAP Java projects loading cds dependency from jar file
+    const cdsEnv: any = cds.env.for('cds', capProjectPath);
+    cds.env = cdsEnv;
```

Comment:
small proposal to avoid `any`
```suggestion
    cds.env = cds.env.for('cds', capProjectPath) as typeof cds.env;
```

Reply:
merged. Thanks